CONTRIBUTING.md

```diff
@@ -2,162 +2,71 @@
 
-> **Install Node.js**
->
-> If you haven't already, you'll need to install Node.js. Use an [official
-> installer](https://nodejs.org/en/download/), or use a utility for managing
-> versions such as [Node Version Manager](https://github.com/creationix/nvm).
-
-Go to the root directory of this repository and
-install its dependencies:
+As an external contributor (not a member of the Palantir org), you will have to fork Blueprint in
+order to contribute code. Clone your fork onto your machine and run the following commands to
+install dependencies:
 
 ```sh
+git clone git@github.com:<username>/blueprint.git # using ssh
 cd blueprint
 npm install
 npm run bootstrap
 ```
 
 ## Developing
 
-Start a new feature branch. We use a format like `[your-initials]/[short-name]`.
-
-For example:
- * `bd/laser-beams`
- * `bd/jittery-cells`
- * `bd/css-class-renames`
-
-To create a new branch for development, run:
-
-```sh
-git fetch
-git checkout -b your-initials/your-branch-name origin/master
-```
-
-
-### Running
-
-Many of the packages have their own compile-and-watch preview tasks which will
-help you confirm that your features are working.
-
-To start the compile-and-watch task, run the default script in the package
-directory.
-
-```sh
-cd packages/table
-npm start
-```
-
-Once the preview server is running, navigate to
-[http://localhost:8080](http://localhost:8080).
-
-
-### Coding
-
-While reviewing your changes, we use
-[`previews`](https://github.com/palantir/blueprint/blob/master/packages/table/preview)
-to demonstrate the functionality of various table features. The relevant package
-previews will also be automatically added to your pull request.
-
-Feel free to modify the examples in the previews to test your code changes. When
-you modify the code in `src/` or `preview/`, the code will be automatically
-recompiled and you can simply refresh to see the result.
-
-
-### Submitting
-
-Once you're satisfied with your changes, you'll need to make sure your code
-will pass code review.
-
-Your code must be **test covered** and **lint free**.
-
-Run all unit tests and measure code coverage:
-
-```sh
-npm run test
-```
-
-If the coverage check fails, you may have to add a new suite of tests or
-modify existing tests in `test/`.
-
-Run code and style linters across all code:
-
-```sh
-npm run lint
-```
-
-These commands must complete successfully, otherwise our CI will automatically
-fail your pull request.
-
-
-### Submit a pull request
-
-Once your code works and the tests and linter pass, you can submit your
-code for review!
-
-```sh
-git add --all .
-git commit
-git push
-```
-
-When committing, write a thorough description of your work so that reviewers
+A typical contributor workflow looks like this:
+
+1. Create a new feature branch. We use a format like `[your-initials]/[short-name]`:
+   `bd/refactor-buttons`.
+1. Run the development server via `gulp` (no arguments).
```

Comment:
Incidentally, package.json include a run target for gulp so you can `npm run gulp` without having to `npm install -g gulp`